### PR TITLE
[monitoring] Add readiness/liveness probe for Prometheus

### PR DESCRIPTION
**Pull request by dulltz**
refer to the link https://github.com/helm/charts/blob/f5bcdc1c4e6c039cfd28184f2ed00f0f38470771/stable/prometheus/templates/server-deployment.yaml#L97-L108

### CODE REVIEW

File: monitoring/base/prometheus/statefulset.yaml [hunk 1]
```diff
@@ -27,6 +27,18 @@ spec:
             - --web.enable-admin-api
           ports:
             - containerPort: 9090
+          readinessProbe:
+            httpGet:
+              path: http://prometheus:9090/-/ready
```
Comment (zoetrope) [inline monitoring/base/prometheus/statefulset.yaml]:
```suggestion
              path: /-/ready
```

File: monitoring/base/prometheus/statefulset.yaml [hunk 2]
```diff
@@ -27,6 +27,18 @@ spec:
...
+              port: 9090
+            initialDelaySeconds: 30
+            timeoutSeconds: 30
+          livenessProbe:
+            httpGet:
+              path: http://prometheus:9090/-/healthy
```
Comment (zoetrope) [inline monitoring/base/prometheus/statefulset.yaml]:
```suggestion
              path: /-/healthy
```

File: monitoring/base/prometheus/statefulset.yaml [hunk 3]
```diff
@@ -27,6 +27,18 @@ spec:
             - --web.enable-admin-api
           ports:
             - containerPort: 9090
+          readinessProbe:
+            httpGet:
+              path: http://prometheus:9090/-/ready
+              port: 9090
+            initialDelaySeconds: 30
```
Comment (zoetrope) [inline monitoring/base/prometheus/statefulset.yaml]: Is it enough?
It takes 10 minutes to recover from a crash.

Comment (dulltz) [reply monitoring/base/prometheus/statefulset.yaml]: That's true! I'll increase `readinessProbe.failureThreshold`.

**Pull request merged**